cypress/integration/step_definitions/backend/ss_subscription.cy.js

```diff
@@ -0,0 +1,32 @@
+import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
+
+let res;
+
+When("I send a GET request to subscription", () => {
+  cy.getSubscription().then((response) => {
+    expect(response.status).to.eq(200);
+    res = response
+  })
+
+});
+
+And("the response body should be correct", () => {
+  const responseBody = res.body;
+
+  expect(responseBody[0].name).to.equal("Free Tier");
+  expect(responseBody[0].package_code).to.equal("free");
```

Comment:
Instead of hard code, try to read expected data from either feature fie or from fixture file